services/bugzilla/bugzilla.service.js

```diff
@@ -13,7 +13,7 @@ const schema = Joi.object({
     .items(
       Joi.object({
         status: Joi.string().required(),
-        resolution: Joi.string().required(),
+        resolution: Joi.string().allow(''),
```

Comment:
I think we still want to have the `.required()` to ensure the field is there. 

Would you mind validing that something like the below still solves the bug?
```suggestion
        resolution: Joi.string().allow('').required(),
```

Reply:
It worked and I updated the PR to `resolution: Joi.string().required().allow('')` to make it look closer to the status property. It works as `resolution: Joi.string().allow('').required()` as well if you want that

Reply:
Awesome, thanks for confirming. IIRC we typically put `.required()` on the end, though as you noted it doesn't make a function difference. 

It's up to you if you'd like to swap those 👍

Reply:
I swapped them and I also squash the commits into a single one. Hope that does not mess up anything.

Reply:
> I swapped them and I also squash the commits into a single one. Hope that does not mess up anything.

Thanks, and no worries. We always squash on merge so feel free to structure your commits in whichever way works best for you/makes the changes easier to review

Reply:
Thanks I am done for this PR now